script.js

```diff
@@ -1,4 +1,83 @@
 /* eslint-disable max-classes-per-file */
+
+// Time code
```

Comment:
- [ ] Please Consider removing disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase.

Reply:
Hello. Tadesse. I would consider removing the linter rule there but it was Microverse's directory that we could disable that specific linter rule [here](https://github.com/microverseinc/curriculum-javascript/blob/17555a4b0793b5065e3aeda6d2568fddd9c45330/books/m1_plain_js_objects_v1_1.md?plain=1#L48-L50). This is part 3 of the same project and therefore, this being one of the issues stopping this PR from being approved, Please advise further on what I should do.